pkg/store/bucket.go

```diff
@@ -237,6 +241,19 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Buckets: []float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120},
 	})
 
+	m.blockSyncs = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "thanos_bucket_store_block_syncs_successful_total",
+		Help: "Total number of successful block syncs.",
+	})
+	m.blockSyncFailures = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+		Name: "thanos_bucket_store_block_syncs_failures_total",
+		Help: "Total number of failed block syncs.",
+	})
+	m.blockSyncLastSuccessful = promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+		Name: "thanos_bucket_store_block_last_successful_sync_time",
```

Comment:
[nit] I think adding `_seconds` at the end help to quickly understand the unit. It's something we did in `thanos_receive_config_last_reload_success_timestamp_seconds` too.

```suggestion
		Name: "thanos_bucket_store_block_last_successful_sync_timestamp_seconds",
```